### DISCUSSION TITLE
borderless vs exclusive mode windows not working properly

### DISCUSSION THREAD

**Discussion by JRGalvan** [rust-windowing/winit#3840] [Q&A] [answered]
Hello!

I am using winit and wgpu to develop a tool. With previous versions of winit I used to struggle to start the window because it will look weird (because of resizing i think), but lately I thought I found a solution, which was to create a default window unfocus and not active, and just when I have everything ready to show to the screen to active the window. But for some reason I activate the window with fullscreen::borderless and the window behaves as fullscreen::exclusive for like 1 min, and then it behaves like borderless, the initial attributes are the default ones. What can I do? The code is really simple, so I can probable share it. 

I am using windows.

**Comment by daxpedda** [comment, marked as answer]
I don't maintain Windows, but this sounds like a bug to me, even though I don't know exactly what "behaves as fullscreen::exclusive" means exactly.

Filing an issue with a minimal reproducible example is probably the way to go.

Apologies for the delay!
